src/cookieconsent.js

```diff
@@ -132,6 +134,24 @@
       // The code "typeof obj === 'object' && obj !== null" allows Array objects
       return typeof obj === 'object' && obj !== null && obj.constructor == Object;
     },
+
+    loadStylesheet: function (stylesheet, complete) {
+      var link = document.createElement("link");
```

Comment:
I don't think it should be used by default. I think by default, you should put the HTML in the <head>, just like any normal person would. I think lazy loading the css by default is a bad idea.

I'd rather remove it than use it by default, because there is no need for it to be used by default. There is however a need for it to be used (if you want to defer loading)

Having said that, if you wanted to defer loading, you can always write a tiny bit of code outside of cookieconsent that says "get this stylesheet, and on complete init cookieconsent".

So yeah, if we don't like it not being used by default, i'd rather remove it